docs/tutorials.md

```diff
@@ -163,7 +163,7 @@
 <!--Arabidopsis Multi-plant Tutorial-->
 <div class="card" style="display:block">
     <a target="_blank" href="" onclick="location.href=this.href.replace('tutorials', 'tutorials/arabidopsis_multi_plant_tutorial');return false;">
-        <img src="https://github.com/danforthcenter/plantcv-binder/blob/master/notebooks/arabidopsis_multi_plant_tutorial/multi_plant_tutorial_card.png?raw=true" alt="Multi-plant tutorial" width="600" height="auto">
+        <img src="https://github.com/danforthcenter/plantcv-tutorial-arabidopsis-tray/blob/main/tutorial_card.png??raw=true" alt="Multi-plant tutorial" width="600" height="auto">
```

Comment:
The image https://github.com/danforthcenter/plantcv-tutorial-arabidopsis-tray/blob/main/tutorial_card.png doesn't exist